app/build.gradle

```diff
@@ -49,7 +49,12 @@ dependencies {
     androidTestImplementation 'com.android.support.test:runner:1.1.0'
     androidTestImplementation 'com.android.support.test.uiautomator:uiautomator-v18:2.1.1'
     androidTestImplementation 'org.hamcrest:hamcrest-library:1.3'
+<<<<<<< HEAD
```

Comment:
You didn't fix this merge conflict. Next time, make sure the build passes before you commit.